spec/test_apps/app/snapshots/books/create/0.json

```diff
@@ -1,7 +1,27 @@
 {
   "request": {
     "method": "POST",
-    "url": "http://www.example.com/books"
+    "url": "http://www.example.com/books",
+    "headers": {
+      "REQUEST_METHOD": "POST",
+      "SERVER_NAME": "www.example.com",
+      "SERVER_PORT": "80",
+      "SERVER_PROTOCOL": "HTTP/1.0",
+      "QUERY_STRING": "",
+      "PATH_INFO": "/books",
+      "HTTPS": "off",
+      "SCRIPT_NAME": "",
```

Comment:
These are not HTTP headers, but something else.

Reply:
See most recent push; these are now more specifically limited to standard CGI headers.